cmd/daemon/start.go

```diff
@@ -49,8 +54,15 @@ func buildStartCmd(parentCmd *cobra.Command) {
 			}
 
 			var password string
+
 			if *passwordOpt != "" {
 				password = *passwordOpt
+			} else if *passwordFromFileOpt != "" {
+				b, err := util.ReadFile(*passwordFromFileOpt)
+				if err != nil {
+					return "", false
```

Comment:
What happens here? Do we show user the error?

Reply:
> What happens here? Do we show user the error? 

We show invalid password, don't need to handle error.

Reply:
No, better we show proper error here, file error and exit